satpy/readers/ahi_hsd.py

```diff
@@ -572,21 +636,25 @@ def read_band(self, key, info):
                               b=float(self.proj_info['earth_polar_radius'] * 1000))
         actual_alt = float(self.nav_info['distance_earth_center_to_satellite']) * 1000 - re
 
+        if self._round_actual_position:
+            actual_lon = round(actual_lon, 3)
+            actual_lat = round(actual_lat, 2)
```

Comment:
The number of digits provided here: are they taken from documentation or have been estimated empirically?

Reply:
Estimated by running with real data and discussions with @simonrp84. Other ideas?

Reply:
No, just curious 😁